### PR TITLE
feat(docs): add jsdocs to the project

### PR DESCRIPTION
**Pull request by RomainGuarinoni**

### CODE REVIEW

File: jsdocs/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            </nav>
+        </div>
+        <div class="core" id="main-content-wrapper">
+            <div class="content">
+                <header class="page-title">
+                    <p></p>
+                    <h1>Home</h1>
+                </header>
+                
+
```
Comment (Guillaume-FAURE) [inline jsdocs/index.html:80]: strange but don't impact the code

File: src/dom-manipulation.ts [hunk 2]
```diff
@@ -52,13 +52,13 @@ export function addNewGameLine(
 
 /**
  * @description Add indicators to the game
- * @param type the type of indicators
- * @param container the container where to put indicators
- * @param number the number of indicators to add
+ * @param {Indicators} type the type of indicators
+ * @param {HTMLDivElement} container the container where to put indicators
```
Comment (Guillaume-FAURE) [inline src/dom-manipulation.ts:56]: for the @param is this really interesting to put the type of the parameters before the name of it, on vscode, the first word is colored, Logically the {} indicate that it's the type we talk about but I don't know the result on the IDE

**Review by Guillaume-FAURE — COMMENTED** [review]
I approve, just strange space in document generated by the jsdoc and question about typescript description of function

File: src/dom-manipulation.ts (hunk 2 above)
Comment (Guillaume-FAURE) [reply src/dom-manipulation.ts:56]: see on vscode it's perfect

File: jsdocs/index.html (hunk 1 above)
Comment (RomainGuarinoni) [reply jsdocs/index.html:80]: this is the code generated by the jsdoc plugin, not ours ahah

File: src/dom-manipulation.ts (hunk 2 above)
Comment (RomainGuarinoni) [reply src/dom-manipulation.ts:56]: Yeah this is how jsdoc knows the type of the @param, see [this](https://jsdoc.app/tags-param.html) !! :D

**Pull request merged**